src/io.rs

```diff
@@ -1266,6 +1266,33 @@ pub trait WriteBytesExt: io::Write {
         self.write_all(&buf)
     }
 
+    /// Writes a sequence of unsigned 32 bit integers to the underlying writer.
+    ///
+    /// # Errors
+    ///
+    /// This method returns the same errors as [`Write::write_all`].
+    ///
+    /// [`Write::write_all`]: https://doc.rust-lang.org/std/io/trait.Write.html#method.write_all
+    ///
+    /// # Examples
+    ///
+    /// Write a sequence of unsigned 32 bit big-endian integers to a `Write`:
+    ///
+    /// ```rust
+    /// use byteorder::{BigEndian, WriteBytesExt};
+    ///
+    /// let src = vec![255, 65284];
+    ///
+    /// let mut wtr = Vec::new();
+    /// wtr.write_u32_from::<BigEndian>(&src).unwrap();
+    /// assert_eq!(&wtr, &[255, 0, 0, 0, 4, 255, 0, 0]);
```

Comment:
These bytes are little endian. Not big endian.

Which makes sense, because the implementation never takes byte order into account. `T` is unused.